packages/web-components/src/styles/patterns/button.ts

```diff
@@ -121,21 +121,24 @@ export const BaseButtonStyles: ElementStyles = css`
   neutralFillActiveBehavior,
   forcedColorsStylesheetBehavior(
     css`
-        :host {
+        :host,
+        :host(.neutral) .control {
```

Comment:
@khamudom can you bind these styles to the appearance attribute here rather than the class.

Reply:
updated